app/src/lib/api.ts

```diff
@@ -506,18 +510,21 @@ export class API {
     try {
       const path = `repos/${owner}/${name}/mentionables/users`
       const response = await this.request('GET', path, undefined, headers)
-      if (response.status === HttpStatusCode.NotModified) {
+
+      if (response.status === HttpStatusCode.NotFound) {
+        log.warn(`fetchMentionables: '${path}' returned a 404`)
         return null
       }
-      if (response.status === HttpStatusCode.NotFound) {
-        log.warn(`fetchAll: '${path}' returned a 404`)
+
+      if (response.status === HttpStatusCode.NotModified) {
+        log.warn(`fetchMentionables: '${path}' returned a 304`)
```

Comment:
Are we sure this deserves a warning? It seems like we'd expect a good amount of 304s from the mentionables endpoint.

Reply:
Nope, 304s are pretty great status codes...